src/main/java/org/jabref/gui/fieldeditors/FieldNameLabel.java

```diff
@@ -3,16 +3,39 @@
 import javafx.geometry.Insets;
 import javafx.geometry.Pos;
 import javafx.scene.control.Label;
+import javafx.scene.control.Tooltip;
 
+import org.jabref.logic.l10n.Localization;
 import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.StandardField;
 
 public class FieldNameLabel extends Label {
+    private Tooltip tip;
 
-    public FieldNameLabel(Field field) {
+    public FieldNameLabel(Field field, String name) {
         super(field.getDisplayName());
 
         setPadding(new Insets(4, 0, 0, 0));
         setAlignment(Pos.CENTER);
         setPrefHeight(Double.POSITIVE_INFINITY);
+        setTip(name);
+    }
+
+    public void setTip(String name) {
+        tip = new Tooltip();
+        if (StandardField.AUTHOR.getName().equals(name)) {
```

Comment:
Is there a reason for testing field.getName() instead of field directly?

Reply:
> I think you could use `field` as an argument for `setTip` and test `field` instead

Thank you for the suggestion. I have changed it.

Reply:
I use if-else here because constant expression is required in switch-case. I cannot write something like:
`case (StandardField.AUTHOR.getName()):`